fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
+    type record_transformer
+    enable_ruby
+    <record>
+      log ${err || msg}
+    </record>
+    remove_keys req,res,msg,name,level,v,pid,err
+  </filter>
   <filter kubernetes.**>
     type flatten_hash
     separator _
   </filter>
+  <filter kubernetes.**>
+    type record_transformer
+    enable_ruby
+    <record>
+      hostname ${kubernetes_host}
+      message ${log}
+      @version 3.1
```

Comment:
Is this version at all significant?
Seems like something that ought to be set as an env var in the Dockerfile.